rbac/management/migrations/0047_outbox.py

```diff
@@ -0,0 +1,35 @@
+# Generated by Django 4.2.15 on 2024-08-12 15:03
+
+from django.db import migrations, models
+import uuid
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        (
+            "management",
+            "0046_remove_workspace_the_combination_of_name_tenant_and_parent_must_be_unique__and_more",
+        ),
+    ]
+
+    operations = [
+        migrations.CreateModel(
+            name="Outbox",
+            fields=[
+                (
+                    "id",
+                    models.UUIDField(
+                        default=uuid.uuid4,
+                        editable=False,
+                        primary_key=True,
+                        serialize=False,
+                    ),
+                ),
+                ("aggregatetype", models.CharField(max_length=255)),
+                ("aggregateid", models.CharField(max_length=255)),
+                ("event_type", models.CharField(db_column="type", max_length=255)),
+                ("payload", models.BinaryField()),
```

Comment:
I am sorry for confusion, we need jsonb here in the end - https://debezium.io/documentation/reference/stable/transformations/outbox-event-router.html#basic-outbox-table

discussed [here](https://redhat-internal.slack.com/archives/C059NUD49PD/p1723550716218259)